subprojects/packagefiles/qarchive/meson.build

```diff
@@ -0,0 +1,79 @@
+project(
+  'QArchive',
+  'cpp',
+  version: '2.1.1',
+  license: 'BSD-3-Clause',
+  default_options: ['cpp_std=c++11'],
+  meson_version: '>=0.49.0')
+
+qt5 = import('qt5')
+
+if (build_machine.system() == 'darwin')
+  # brew installs into this path
+  libarchive_dep = dependency('libarchive',
+    default_options: 'pkg_config_path=/usr/local/opt/libarchive/lib/pkgconfig')
```

Comment:
Huh, is this not set up by brew already?

Reply:
This should be added to ci_config.json , not here.

Reply:
Actually I don’t know if libarchive even has a pkgconfig file…

Reply:
It has at least on Linux…